openassessment/xblock/test/test_staff_area.py

```diff
@@ -55,6 +55,10 @@ class NullUserService(object):
     def get_anonymous_user_id(username, _):
         return username
 
+    @staticmethod
+    def get_current_user():
+        return MagicMock(opt_attrs={})
```

Comment:
Note: This is to prevent crashing during test runs, as otherwise this `NullUserService` is not fully implemented and will crash when asking for the current user's timezone.